CONTRIBUTING.md

```diff
@@ -4,101 +4,123 @@ Contribuições sempre serão bem vindas, sejam pequenas ou grandes. Leia o [có
+- Proibido publicar soluções de atividades avaliativas das disciplinas que se repetem todos os períodos, tais como os roteiros de LEDA e os laboratórios de LP2.
+
+## Indicações
+
+- Seja claro nos seus commits.
+- Caso esteja tendo dificuldades de trabalhar com o Git, contate alguém da equipe e peça ajuda. 
+- Caso vá realizar alguma implementação, respeite os padrões da comunidade e produza um código legível.
+- Sempre que for mudar algo, verifique se é necessário alterar algo na documentação. É importante que tudo esteja atualizado.
+- Ao produzir materiais textuais (como resumos), dê preferência a arquivos *.md*. Arquivos *.pdf* e *.doc* tornam o nosso repositório muito pesado.
+
+## Restrições de Upload
+
+### Material de Terceiros
+
+#### O que posso referenciar (linkar)?
+
+Qualquer material útil que esteja **publicamente** disponível na internet pode ser referenciado. Se os materiais de uma disciplina são mantidos disponíveis em um site, referencie-o no `linksUteis.md`.
+O link de arquivo no Google Drive não é publicamente disponível, tá? (:
+
+#### O que posso upar?
+
+Essa é uma pergunta mais difícil. Prioritariamente, o Tamburetei quer disponibilizar **provas** e **listas de exercícios** de períodos passados. Em alguns casos particulares, outros tipos de materiais, como slides, também são permitidos.
+
+No entanto, por exceção das provas (que se tornam documentos públicos), esses materiais podem estar protegidos por *copyright*. Então, se o material que você deseja upar tem o nome de um professor, precisaremos da autorização dele antes, certo?
+
+**Ainda não tem certeza se pode upar?** Consulte um de nossos membros e esclareceremos sua dúvida!
+
+### Tipos de Arquivo
+
+Visando manter o repositório leve e respeitar as limitações da hospedagem gratuita no GitHub, a equipe do Tamburetei fará o possível para manter apenas **markdowns** *(.md)* e **arquivos de código** *(.py, .java, etc)* em nosso repositório. Por isso, arquivos muito grandes ou com outras extensões serão movidos para nosso Google Drive e referenciados aqui através de links.
 
-Se você escolheu uma issue para contribuir, não esqueça de avisar lá que você está trabalhando naquilo. Se é algo que ainda não exista uma issue, crie! É importante notificar o que você está fazendo.
+`Essa restrição não afeta as contribuições, apenas informa que arquivos aqui hospedados poderão ser transferidos posteriormente.`
 
-### Clonar o repositório
 
-Após isso, você deve dar um fork do projeto. Existe um botão na página do github para isso. Quando ele terminar de realizar o fork, você deve copiar a url do repositório do fork e clonar na sua máquina. 
+## Como contribuir?
 
+Se você escolheu uma issue para contribuir, não esqueça de avisar lá que você está trabalhando nela. Se que contribuir com algo que ainda não foi citado em uma issue, crie-a! É importante notificar o que você está fazendo.
```

Comment:
Acho que seria legal colocar mais uma espécie de ênfase aqui, para que todos realmente tenham noção de que isso é muito importante.

Reply:
De que forma? Ênfase no sentido de negrito ou de mais texto?

Reply:
De negrito ou itálico.

Reply:
Corrige pra mim? ❤️

Reply:
Sim. 💘